doc/tietomalli.md

```diff
@@ -1,51 +1,90 @@
+| NodePort                              | layout.node_port                   | Graafin solmun portti (kiinnityskohta)                        | Jos solmussa yhdistyy kaksi eri käsitettä (yleensä vaihdepistettä) portit kuvaavat suuntia joista jänne voi siihen kytkeytyä |
+| LayoutSegment (sama luokka kuin yllä) | layout.edge_segment                | Graafin jänteen geometrian (keskilinjan) segmentti            | Geometriaviivan pätkä: metatiedoiltaan yhtenevä osa pisteviivaa                                                              |
+| **Muut käsitteet**                    |                                    |                                                               |                                                                                                                              |
+| PVDocument                            | projektivelho.document             | Projektivelhon dokumentti                                     | Projektivelhosta ladattu yksittäinen tiedosto                                                                                |
+| SwitchStructure                       | common.switch_structure            | Vaihdetyyppi                                                  | Vaihdetyypin kuvaus ja sen RATO-määrityksen mukainen rakenne                                                                 |
+| SwitchOwner                           | common.switch_owner                | Vaihteen omistaja                                             |                                                                                                                              |
+| LocationTrackOwner                    | common.location_track_owner        | Sijaintiraiteen omistaja                                      |                                                                                                                              |
+
+## Sijaintien esitysmuodot
+
+### Koordinaatit
+
+Geoviitteessä koordinaatit ovat tyypillisesti määräävä esitysmuoto sijainnille ja muut muodot ovat johdettuja niistä.
+Koordinaatteja käsitellään useassa eri koordinaatistossa, erityisesti suunnitelmien tarkoissa geometrioissa.
+Paikannuspohjassa koordinaatit ovat yhtenäistetty ETRS-TM35FIN koordinaatistoon.
+
+### m-arvot
+
+Geoviitteessä käytetään m-arvoja (matka metreinä) kuvaamaan sijaintia tiettyä geometriaa pitkin. M-arvo on siis
+etäisyys ko. geometrian (raiteen, pituusmittauslinjan, ...) alusta. Koska tulkinta on riippuvainen kontekstista,
+tulee m-arvojen kanssa aina huomioida minkä käsitteen m-arvosta puhutaan. Esimerkiksi yhdellä raiteen sijainnilla
+on erilliset m-arvot suhteessa sen segmentteihin (segmentM) tai raiteen koko pituuteen (alignmentM).
+
+### Rataosoitteet
+
+Rataosoitteet on RATO-ohjeistuksen mukaisesti lasketut osoitteet rataverkon sijainneille, muotoa KM+m, esim 0123+1234.
+Tyypillisimmin osoitteista käytetään vain tasametrejä, mutta niille voidaan antaa myös tarkempi arvo metrin desimaaleina.
```

Comment:
Jos tätä miettii Geoviitteen näkövinkkelistä niin tämä lause on mielestäni vähän jännä, sillä meillä millintarkka muoto tuntuu olevan enemmänkin sääntö kuin poikkeus (varsinkin UI:lla). Tarkoitetaanko tällä sitä, että muut järjestelmät käyttäisivät pääasiassa tuota desimaalitonta muotoa ja että me pääosin annetaan niitä ulos? If so, muotoilisin tämän enemmän ehkä siihen suuntaan

Reply:
Hmm. Meillä on puhtaasti toteutusteknisesistä syistä ne millintarkat "oletus" sillä laskentahan on niin tarkkaa kun halutaan olevan ja tasametri on siitä vinkkelistä erikoistapaus. Mutta rato, erinäiset kaaviot ja muut järjestelmät puhuu lähinnä tasametreistä koska ei tuolla osoitteistolla juuri tarkempaan oikeasti pysty. Eli kyllä tämä lause on minusta ihan paikkansa pitävä. Yleisesti (as in yleisesti ratakäsitteistössä) puhutaan vain tasametreistä mutta on myös määritelty tapa esittää noita tarkempia. Kaippa tuon ratakäsitteistössä-sanan voisi tuohon lisätä.